docs/connecting-data/connecting-to-apis-and-external-data/harmonic-api.md

```diff
@@ -1,9 +1,99 @@
 ---
-description: How to Connect DataDistillr to the Harmonic API
+description: How to Connect DataDistillr to the Harmonic.ai API
 ---
 
-# Connecting to Harmonic
+# Connecting to Harmonic.ai
```

Comment:
Leave this as Harmonic - all their icons, etc. seem to be just Harmonic, not Harmonic.ai